### ISSUE TITLE
Web accessibility

### ISSUE THREAD

**Issue by brianamarie** [FreeCodeCamp/hikes#164]
Initially proposed by OMahonyD

Series of videos covering web accessibility as it pertains to users, designers, and developers.

(1) intro to Web access (2) why Web access (3) visual (4) auditory (5) cognitive (6) motor

**Comment by hallaathrad** [comment]
where's the "like" button!!!!?

**Comment by brianamarie** [comment]
Haha! We'll have to get GitHub on creating that feature...

**Comment by OMahonyD** [comment]
#Introduction

Today we are going to talk about web accessibility and what it means to build inclusive applications and websites. 

So if we were to type the word "accessibility" into Google we might get the definition:

"The design of products, services and environments for people with disabilities."

If we extend that definition into the area of the *Internet* we might arrive at the phrase:

"Designing your site or application in such a way so that people with disabilities can perceive, understand, navigate and interact with it easily and enjoyably."

While this is true, accessible and inclusive design is more about creating something that can be used by everyone.

The web is fundamentally designed to work for all people, whatever their hardware, software, language, culture, location, a
and also their physical and mental ability.

By incorporating certain techniques in our code and being mindful of best practice guidelines in our designs we can truly contribute to an inclusive Web experience for everybody.

**Comment by brianamarie** [comment]
@OMahonyD I think this is a great starting point! I would include an overview of the videos to come: visual, auditory, cognitive, and motor. Also, if the "why" video content is equally as concise, we could include the "why" video content in this same video here as well. What do you think?

**Comment by OMahonyD** [comment]
@brianamarie Updated script there. I'll get working on the next one and we'll see how concise it becomes. Thanks for your advice. D

**Comment by brianamarie** [comment]
This looks **great**, @OMahonyD! :+1: 

When you get ready to record, make sure you have lighting, whiteboard/blackboard positioning and writing, and editing strategy all figured out. If you have any questions, feel free to tag me here or message me on gitter. :) Best of luck! I can't wait to see the video!

**Comment by OMahonyD** [comment]
#Why Should We Care About Web Accessibility?

Following on from our introduction, we spoke about making our sites and applications Perceivable, Operable, Understandable and Robust. This means trying to create the perfect accessible digital experience:

Usable by everyone ...
... on any user agent (software)
... on any kind of device
... with any speed of connection
... in any environment

And the glaring question is: Why? Why should we care?

There are a huge number of reasons but we're going to focus on 4 main ideas.

Number 1:
There are just so many of us. There are around 7 billion people on the planet today and it's estimated that between 10 and 20% of those are living with a disability. Right now in 2016, the Internet remains a mystery for 70% of the world's population. But this is all about to change. By the end of this decade the majority of individuals will be online.

Number 2:
Search Engines.
[to be added]

Number 3:
Human Resources
[to be added]

Number 4:
Regulation.
[to be added]

**Comment by brianamarie** [comment]
@OMahonyD Can't tell you how excited I am about these! Any work on the videos yet?

**Comment by OMahonyD** [comment]
Hi Briana,So sorry for the delay. I've been away for a while and only back home today. I've tried recording with phone but kids and dogs providing cameo appearances make for unprofessional viewing. I'm going to try hard to get something done soon maybe using my Web cam on a very old mac. I'll keep in touch.Dan


Sent from my Samsung Galaxy Note 4 - powered by Three

-------- Original message --------

**Comment by brianamarie** [comment]
@OMahonyD If you would like to take this over again, I'd be :sparkles: thrilled! 😄  If not though, I understand. I'm going to add 'help wanted' tag for now - please feel free to let me know if you'd like to contribute in this way again.

**Comment by brianamarie** [comment]
### So you want to help with this issue? Great! Keep track of your steps here. 

- [ ] Check out these resources to get ready
  - [ ] [Article with general guidelines](https://medium.freecodecamp.com/let-s-go-for-a-hike-here-s-your-map-4301df6e3801#.f91m4eq85)
  - [ ] [Video challenge chat](https://gitter.im/FreeCodeCamp/hikes)
  - [ ] [Hike repository](https://github.com/FreeCodeCamp/hikes)
- [ ] Find this topic in the [video challenge tracking document](https://github.com/FreeCodeCamp/hikes/blob/master/video-challenge-tracking.md). If it's not there, add it where you think it fits best.
- [ ] Make a pull request to assign yourself as an assignee in the [video challenge tracking document](https://github.com/FreeCodeCamp/hikes/blob/master/video-challenge-tracking.md).
- [ ] Propose a script as a [markdown file](https://guides.github.com/features/mastering-markdown/) in a Pull Request
  - Create a new branch with `<yourusername-videoname>`
  - Create a new file labeled `<video-title.md>` within the appropriate folder (`scripts/<unit-title>/<video-title.md>`). If that folder doesn't exist yet, create one!
  - Save your changes and open a pull request comparing to master. `@` mention brianamarie for feedback.
- [ ] Record your video
  - Set up a whiteboard and record your video 
- [ ] Comment in your PR with the link to your video. `@` QuincyLarson and brianamarie
  - You'll get feedback, and we'll work with you to help refine the video so we can include it on FreeCodeCamp.com
- [ ] Open a PR in the [Free Code Camp repository](https://github.com/freecodecamp/freecodecamp) including your video links, scripts, and questions to get them added to the site

**Comment by OMahonyD** [comment]
Hi Briana,
Sure thing. I have just had the busiest year so far and haven't had a
chance to keep writing the scripts. It had to be put on the back foot as
other priorities took precedence. That's no problem at all - the more
people working on something so important as accessibility the better. :) I
have a number of other scripts that haven't been finalised yet but will
throw them up to github and others may find them useful too. If I get
around to recording, I'll let you guys know. Have a great start to the
summer.
Dan



[image: --]

Daniel O'Mahony
[image: https://]about.me/danomah
<https://about.me/danomah?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>

**Comment by brianamarie** [comment]
@OMahonyD Thanks for letting us know! If you have time again to come back and help, we'd love to have you. 😄

**Comment by QuincyLarson** [comment]
We are including this topic in our core curriculum so I'm closing the video challenge issue.
